flow/compositor_context.h

```diff
@@ -23,18 +23,19 @@ namespace flutter {
 
 class LayerTree;
 
+// The result status of CompositorContext::ScopedFrame::Raster.
 enum class RasterStatus {
   // Frame has been successfully rasterized.
   kSuccess,
-  // Frame is submitted twice. This is only used on Android when
-  // switching the background surface to FlutterImageView.
+  // Frame has been submited, but should be submitted again. This is only used
```

Comment:
Here and below, avoid using `should`.

Instead, describe what `must`/`must not` happen to make it clearer about what developers reading this code can actually expect.